### ISSUE TITLE
create dataset

### ISSUE THREAD

**Issue by haithamRD** [rishabh-karmakar/Facial-Recognition-Based-Attendance-System#3]
hey sir ,
i m so sorry for derengement , i have one probleme when i run Add_New.py the camera not show for create data can i u help please

**Comment by rishabh-karmakar** [comment]
In that case, I request you to check all the dependencies in testing.txt and specially the versions of numpy and tensorflow. Also, try running Add_New.py separately. Depending on your ram and cache, it may or may not take some time for the camera module to load up. And if there's some error coming in terminal, feel free to share the screenshot here.

**Comment by haithamRD** [comment]
all dependencies is right .  i try to run this page but the camera not loaded. i have this probleme on 2 os on ubuntu and windows

**Comment by rishabh-karmakar** [comment]
That... isn't right. Might I suggest opening up a Jupyter notebook and see if the camera is indeed opening or not? 
Also, this problem is in just Add_New or in Recog.py as well. Try running that once.

**Comment by rishabh-karmakar** [comment]
Run this to check if camera is working.
```
import numpy as np
import cv2

cap = cv2.VideoCapture(0)
while(True):
    # Capture frame-by-frame
    ret, frame = cap.read()

    # Our operations on the frame come here
    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)

    # Display the resulting frame
    cv2.imshow('frame',gray)
    if cv2.waitKey(1) & 0xFF == ord('q'):
        break

# When everything done, release the capture
cap.release()
cv2.destroyAllWindows()
```

**Comment by haithamRD** [comment]
this problem just in Add_New.py

**Comment by haithamRD** [comment]
i test the code on jupyter notebook but i have a same probleme  
pliz help
![Capture1](https://user-images.githubusercontent.com/34986651/83898094-598e3100-a74e-11ea-868e-9487c7d3c2c0.PNG)

**Comment by haithamRD** [comment]
tit 's blocked in this point 
![Capture](https://user-images.githubusercontent.com/34986651/83898470-db7e5a00-a74e-11ea-818b-73bfc452fd2c.PNG)

**Comment by haithamRD** [comment]
when i verifed the code the problem in detect_face function

**Comment by rishabh-karmakar** [comment]
Okay, in that case; try the following. 
1. Download the model again. - model-20170512-110547.ckpt-250000
2. Can you print a few statements in the face_feature file and get me which function/ line is actually causing the error.
Most problem originate from the change in tensorflow version. Many functions were deprecated.

**Comment by haithamRD** [comment]
okey , can i  get the perfect running code? you can send to me in my email pliz 
haitham.rdina@gmail.com

**Comment by haithamRD** [comment]
tensorflow version is 1.15.0
i try the download the model and i had the same problem

**Comment by rishabh-karmakar** [comment]
So I could assume Recog.py and the initial unlock Auth1.py is working and camera is opening?

**Comment by haithamRD** [comment]
yes exactly the camera opens in recog.py and Auth.py but it doesn't loaded the time to create data in Add_New.py if u can get a perfect running program please

**Comment by haithamRD** [comment]
if you have a documentation about this app explain how to use it please because i test all features and dependencies for use it

**Comment by rishabh-karmakar** [comment]
Again, I made this way ago and I just tested it, again and Im not facing similar error, so Im not too sure how can I help you;
What I would recommend is use this code and deploy it on backend interfaces like flask, django and like.

**Comment by adityafaadil** [comment]
@rishabh-karmakar hey sir! can I get the download link for 'model-20170512-110547.ckpt-250000'?

**Comment by hafsaabid416** [comment]
yeah, the google drive link doesn't have the model
